donkeycar/parts/actuator.py

```diff
@@ -166,60 +180,6 @@ def set_pulse(self, pulse):
     def run(self, pulse):
         self.set_pulse(pulse)
 
-class VESC:
```

Comment:
I rebased against main.  For some reason it shows me deleting the VESC class.  What is up with that?

Reply:
I just copied it back in from main to fix this issue.